azure-kusto-data/azure/kusto/data/request.py

```diff
@@ -356,23 +345,22 @@ def execute_streaming_ingest(
             request_params["mappingName"] = mapping_name
 
         request_headers = {
-            "Accept-Encoding": accept_encoding,
-            "Connection": connection,
+            "Accept": "application/json",

```

Comment:
Consider having a default dictionary that is common fro both streaming ingestion and executing queries.